st2actions/st2actions/config.py

```diff
@@ -44,7 +44,19 @@ def _register_common_opts():
 def _register_action_runner_opts():
     default_python_bin_path = sys.executable
     base_dir = os.path.dirname(os.path.realpath(default_python_bin_path))
-    default_virtualenv_bin_path = os.path.join(base_dir, 'virtualenv')
+
+    custom_virtualenv_binary = '/usr/local/bin/virtualenv'
```

Comment:
Also, yes, I'm aware this won't work on Windows (well, it would actually fall back to else clause so it might work), but that's not our problem, because:

1. We currently don't support Windows as a host operating system for StackStorm and I don't think we will do it any time soon either (we already have tons and tons of issues with getting stuff to work consistently and reliably across supported Linux distros so supporting yet another totally different OS is nothing but wishful thinking)
2. Even if we ever supported Windows, there are many other things we would need to change before this would become a problem (e.g. a lot of things assume "/opt/stackstorm" path right now, etc.).